### PR TITLE
postsubmit-batch endpoint

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**

### CODE REVIEW

File: proto/api.proto [hunk 1]
```diff
@@ -660,10 +672,35 @@ message PostSubmitRequest {
   bool skipPreFlight = 2;
 }
 
+message PostSubmitBatchRequestEntry {
```
Comment (aspin) [inline proto/api.proto]: Maybe just reuse `PostSubmiRequest` instead of creating this struct?

File: proto/api.proto [hunk 2]
```diff
@@ -660,10 +672,35 @@ message PostSubmitRequest {
...
+  SubmitStrategy SubmitStrategy = 2;
+}
+
 message PostSubmitResponse {
   string signature = 1;
 }
 
+message PostSubmitBatchResponseEntry {
+  string signature = 1;
+  string error = 2;
```
Comment (aspin) [inline proto/api.proto:695]: perhaps also a boolean that indicates if it's submitted or not? could encapsulate that in error as well –– wdyt?

**Review by aspin — APPROVED** [review]

File: proto/api.proto (hunk 2 above)
Comment (tinystarinagalaxy) [reply proto/api.proto:695]: i think a boolean is a better idea. makes the client to figure out if it was submitted instantly rather checking error messages for something that is not an error

File: proto/api.proto (hunk 1 above)
Comment (tinystarinagalaxy) [reply proto/api.proto]: ok done

**Review by aspin — APPROVED** [review]

**Pull request merged**